src/screens/OrganizationActionItems/OrganizationActionItems.spec.tsx

```
@@ -53,644 +64,765 @@ const debounceWait = async (ms = 300): Promise<void> => {
+      <MemoryRouter initialEntries={[`/orgactionitems/${orgId}`]}>
+        <Provider store={store}>
+          <LocalizationProvider dateAdapter={AdapterDayjs}>
+            <I18nextProvider i18n={i18n}>
+              <Routes>
+                <Route
+                  path="/orgactionitems/:orgId"
+                  element={<OrganizationActionItems />}
+                />
+                <Route
+                  path="/"
+                  element={<div data-testid="paramsError"></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)